source/dsl/DangerDSL.ts

```diff
@@ -23,9 +23,8 @@ export interface DangerDSLType {
 export class DangerDSL {
   public readonly github: Readonly<GitHubDSL>
 
-  constructor(pr: GitHubPRDSL, public readonly git: GitDSL) {
-    this.github = {
-      pr
-    }
+  constructor(platformDSL: any, public readonly git: GitDSL) {
+    // As GitLab etc support is added this will need to be changed
+    this.github = platformDSL
   }
```

Comment:
This object is the root of what we expose as "danger" inside a Dangerfile, so whatever it's shape is ends up being the API for consumers. So I can't really rename it ATM